.github/workflows/codeql.yml

```
@@ -270,7 +270,7 @@ jobs:
         from pathlib import Path
 
         # Find the plugin directory that contains the CodeQL plugin
-        plugin_dir = list(Path(os.environ['GITHUB_WORKSPACE']).rglob('.pytool/Plugin/CodeQL'))
+        plugin_dir = list(Path(os.environ['GITHUB_WORKSPACE']).rglob('Basetools/Plugin/CodeQL'))

```

Match case sensitive to avoid potential OS-specific issues.

```suggestion
        plugin_dir = list(Path(os.environ['GITHUB_WORKSPACE']).rglob('BaseTools/Plugin/CodeQL'))
```

---

Fixed.